test/test_utils.py

```diff
@@ -954,6 +954,91 @@ def test_escape_url(self):
         )
         self.assertEqual(escape_url('http://vimeo.com/56015672#at=0'), 'http://vimeo.com/56015672#at=0')
 
+    def test_js_to_json_vars_strings(self):
+        def parse_js_to_json(js, vars):
+            try:
+                return json.loads(js_to_json(js, vars))
+            except json.JSONDecodeError as e:
+                raise ValueError(f'JSON: {e.msg}: line {e.lineno} column {e.colno} "{e.doc}"')
```

Comment:
why the throw and catch?

Reply:
For line and column numbers with the JSON document, it's not necessary, just nice to have if the test breaks.